lib/wit/deps/fastly/compute.wit

```diff
@@ -711,6 +711,110 @@ interface kv-store {
   ) -> result<_, error>;
 }
 
+/*
+ * Fastly KV Store
+ */
+interface kv-store {
```

Comment:
Is there any sort of deprecation tag (or even a comment) we can put here?

Reply:
There is, but we could instead just remove the `object-store` interface and polyfill it in the adapter with the new `kv-store` import. Either way, let's hold off until the `kv-store` interface has been filled out :)